sources/packages/web/src/components/layouts/FullPageContainer.vue

```diff
@@ -1,39 +1,41 @@
 <template>
-  <v-container :fluid="fullWidth">
-    <header class="mb-2">
+  <div :class="noSidebar ? 'no-sidebar' : ''">
```

Comment:
I didn't get why are we using this style. (margin-inline-start:256px)

Reply:
I have already added the PR description. For these types of pages with no sidebar, the full page container should only take the space after sidebar space.( full page container should not use sidebar space, even if the sidebar is not there)
![image](https://user-images.githubusercontent.com/77353155/186254923-4ae44e2c-9b49-49c6-9d53-4f6f4318e38e.png)
![image](https://user-images.githubusercontent.com/77353155/186255089-3c0a412b-3ec0-4d6b-9c90-08113ff3bda7.png)
![image](https://user-images.githubusercontent.com/77353155/186258486-f804abe6-5fe7-457d-84a0-dd3360a5b513.png)

Reply:
Thanks for explaining. I get what is happening with the style, but what is the user experience advantage here ? @hellolynn-tbtb can you please help me understand this?

Reply:
Making a note @dheepak-aot and I chatted and got the answers, which connects to an earlier chat I had with @ann-aot :)